hub/core/dataset/dataset.py

```diff
@@ -378,27 +391,27 @@ def delete_tensor(self, name: str, large_ok: bool = False):
 
         Raises:
             TensorDoesNotExistError: If tensor of name `name` does not exist in the dataset.
-            InvalidTensorNameError: If `name` is in dataset attributes.
         """
         auto_checkout(self)
         name = name.strip("/")
 
         while "//" in name:
```

Comment:
these lines from name.strip to full_name are repeated in create_tensor and rename, can be taken out.